app/assets/javascripts/miq_application.js

```diff
@@ -734,6 +734,23 @@ function miqChartMenuClick(itemId) {
   }
 }
 
+function miqRESTAjaxButton(url, button) {
+  var form = $(button).parents('form:first')[0];
+  if (form) {
+    $(form).submit(function(e) {
+      e.preventDefault();
+      return false;
+    });
+    miqJqueryRequest(form.action, {
+      beforeSend: true,
+      complete: true,
+      data: $(form).serialize()
```

Comment:
The ```edit``` routes are now working correctly.
So other than the ```form serialize``` issue that I mentioned earlier, everything else seems to be working well.